src/utils_general/statistics.py

```diff
@@ -92,3 +92,12 @@ def get_return_period_function_empirical(df_rp: pd.DataFrame, rp_var: str):
     df_rp["exceedance_probability"] = df_rp["rank"] / (n + 1)
     df_rp["rp"] = 1 / df_rp["exceedance_probability"]
     return interp1d(df_rp["rp"], df_rp[rp_var])
+
+
+def calc_mpe(observations: np.array, forecast: np.array) -> float:
```

Comment:
Maybe you can add a docstring, like in the functions above?

Reply:
(and btw the other docstring formats are a bit messed up from the Black / flake8 formatting, and I didn't catch it in the PR ><. Each `param` should be on a new line)

Reply:
Oh no! good catch, apparently the tool I used to wrap them messed this up. Fixed them here, but didn't feel like fixing all of them 😅